Gems/Atom/RPI/Code/Source/RPI.Public/Scene.cpp

```diff
@@ -1116,8 +1116,9 @@ namespace AZ
                                 for (; index < size; ++index)
                                 {
                                     PipelineStateData stateData = pipelineStateList[index];
-                                    if (stateData.m_multisampleState == multisampleState
-                                        && stateData.m_renderAttachmentConfiguration == renderAttachmentConfg)
+                                    if (stateData.m_multisampleState == multisampleState &&
```

Comment:
Do we want this behavior across all the passes? Why not set all the shadowmap-passes to clear instead of only the first one?

Reply:
This behavior should be fine for all passes, since the clear-action is ultimately not taken from this pipeline-state.

The shadows are a bit confusing, partially because the same ShadowMap - Pass is used for both cascaded and projected Shadow maps in slightly different configurations.
As far as i can tell, the clear - Action clears the entire array-slice, which is fine for projected shadowmaps where each cube-map gets it's own array slice.
But for cascaded shadowmaps multiple cascades are stored in one array slice, and then the clear action would also clear previous cascades

Reply:
added a comment.